inc/admin/profile/namespace.php

```diff
@@ -17,8 +17,14 @@ function bootstrap() {
 	add_action( 'all_admin_notices', __NAMESPACE__ . '\\output_profile_messages' );
 	add_action( 'personal_options_update',  __NAMESPACE__ . '\\handle_revocation', 10, 1 );
 	add_action( 'edit_user_profile_update', __NAMESPACE__ . '\\handle_revocation', 10, 1 );
+	register_scripts();
 }
 
+function register_scripts() {
+	wp_register_script( 'oauth2-edit-application', plugins_url( '../../../assets/js/oauth-admin.js', __FILE__ ) );
```

Comment:
Instead of using a relative path here, we should set a `const PLUGIN_FILE = __FILE__` in the main plugin file, then use `plugins_url( 'assets/js/oauth-admin.js', OAuth2\PLUGIN_FILE )` here.